src/components/scenes/CreateWalletImportScene.js

```diff
@@ -71,23 +71,31 @@ export class CreateWalletImportComponent extends Component<CreateWalletImportPro
   }
 
   render () {
-    const { error, isProcessing } = this.state
+    const { error, isProcessing, input } = this.state
+    const { selectedWalletType } = this.props
+    const { currencyCode } = selectedWalletType
+    const specialCurrencyInfo = getSpecialCurrencyInfo(currencyCode)
+    if (!specialCurrencyInfo.isImportKeySupported) throw new Error()
+    const instructionKey = specialCurrencyInfo.isImportKeySupported.privateKeyInstructions
+    const instructionSyntax = s.strings[instructionKey]
```

Comment:
This is unnecessarily complex. Have privateKeyInstructions just be the actual string you want to display. Same for privateKeyLabel.